### DISCUSSION TITLE
Welcome to proxmox-helper-scripts Discussions!

### DISCUSSION THREAD

**Discussion by bramsuurdje** [BramSuurdje/proxmox-helper-scripts#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Drallas** [comment]
This frontend is much better imho! Any feedback received from tteck?

**Reply by bramsuurdje** [reply]
Thank you! I really appreciate it, but no, he isn't interested.

**Reply by Drallas** [reply]
Bad luck, your version is much nicer and easier to use. 

btw after I donated to tteck’s project, PayPal confirmed it went to Angela T………. So tteck seems to be a she.
